### PR TITLE
Interactivity

### PR DESCRIPTION
**Pull request by shamansir**
Please do not merge yet, I plan some minor additions, but it's ok to review.

### CODE REVIEW

File: publish.js [hunk 1]
```diff
@@ -139,7 +139,8 @@
 
             utils.forcedJS('//' + playerDomain + '/' + PLAYER_VERSION_ID + '/bundle/animatron.min.js',
                 function () {
-                    anm.Player.forSnapshot(TARGET_ID, snapshotUrl, anm.importers.create('animatron'));
+                    anm.Player.forSnapshot(TARGET_ID, snapshotUrl, anm.importers.create('animatron'),
+                    window.actions ? window.actions : null);
```
Comment (mcm69) [inline publish.js]: just `window.actions` as I said before

File: src/anm/animation/element.js [hunk 2]
```diff
@@ -1115,6 +1115,66 @@ Element.prototype.bounce = function(nrep) {
...
+ * Will pause _only_ for the time where element is "alive", i.e. if current time is
+ * outside of its band, element won't render instead. Also, no band `START`/`STOP` events
+ * will be fired in any case.
+ *
+ * See also: {@link anm.Element#band band()}, {@link anm.Element#jump jump()}, {@link anm.Element#unfreeze unfreeze()}.
+ *
+ * @return {anm.Element} itself
+ */
+Element.prototype.freeze = function() {
+    if (this.freezed) return this;
```
Comment (mcm69) [inline src/anm/animation/element.js]: how about `.frozen`?

File: src/anm/loader.js [hunk 3]
```diff
@@ -14,10 +14,28 @@ var Animation = require('./animation/animation.js');
...
+    // assign
+    player.anim = anim;
+    if (callback) callback.call(player, anim);
+};
+
 Loader.loadFromUrl = function(player, url, importer, callback) {
     if (!JSON) throw errors.system(ErrLoc.S.NO_JSON_PARSER, player);
 
-    mporter = importer || anm.importers.create('animatron');
+    importer = importer || anm.importers.create('animatron');
```
Comment (mcm69) [inline src/anm/loader.js]: lolol

Comment (shamansir) [inline src/anm/loader.js]: \_(ツ)_/¯

File: publish.js (hunk 1 above)
Comment (shamansir) [inline publish.js]: true, will fix it, thanks

File: src/anm/animation/element.js (hunk 2 above)
Comment (shamansir) [inline src/anm/animation/element.js]: :+1:

**Comment by shamansir** [comment]
I will add things I wanted later. There's a masks fix in this PR, so be it merged.

**Pull request merged**